ui/src/dashboards/components/DashboardImportOverlay.tsx

```diff
@@ -0,0 +1,83 @@
+// Libraries
+import React, {PureComponent} from 'react'
+import {withRouter, WithRouterProps} from 'react-router'
+import _ from 'lodash'
+import {connect} from 'react-redux'
+
+// Actions
+import {getDashboardsAsync} from 'src/dashboards/actions/v2'
+import {createDashboardFromTemplate as createDashboardFromTemplateAction} from 'src/dashboards/actions/v2'
+
+// Types
+import ImportOverlay from 'src/shared/components/ImportOverlay'
+import {AppState, Organization} from 'src/types/v2'
+
+interface DispatchProps {
+  createDashboardFromTemplate: typeof createDashboardFromTemplateAction
+  populateDashboards: typeof getDashboardsAsync
+}
+
+interface StateProps {
+  orgs: Organization[]
+}
+
+interface OwnProps extends WithRouterProps {
+  params: {orgID: string}
+}
+
+type Props = OwnProps & StateProps & DispatchProps
+
+class ImportDashboardOverlay extends PureComponent<Props> {
```

Comment:
should this be DashboardImportOverlay to match the file name

Reply:
thank you !